examples/notebooks/README.md

```diff
@@ -0,0 +1,12 @@
+# Intel(R) Extension for Scikit-learn* notebooks :snake:
+
+This folder contains examples of jupyter notebooks which using Intel(R) extension for scikit-learn for known small datasets.  
+You need jupyter notebook to run these .ipynb files:
+
+```bash
+conda install -c conda-forge notebook
+```  
+or  
+```bash
+pip install notebook
+```  
```

Comment:
I think cool to add table with current examples and small description

Reply:
For example
| **Algorithm** | **Workload** | **Notebook**|
|----------------|-----------------|----------------|
| **Logistic Regression** |  CIFAR-100 | [View source on GitHub](https://github.com/intel/scikit-learn-intelex/blob/master/examples/notebooks/logistictic_regression_cifar.ipynb) |
| **SVC**                         |  Adult         |     [View source on GitHub](https://github.com/intel/scikit-learn-intelex/blob/master/examples/notebooks/svc_adult.ipynb) |
| ....                            | ....               | ....                |